packages/ui-components/src/DropDown.tsx

```diff
@@ -0,0 +1,103 @@
+/*
+ * Copyright 2018-2020 IBM Corporation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { FormGroup, MenuItem } from '@blueprintjs/core';
+import { ItemPredicate } from '@blueprintjs/select';
+import { Select, Button } from '@jupyterlab/ui-components';
+import * as React from 'react';
+
+const DROPDOWN_ITEM_CLASS = 'elyra-form-DropDown-item';
+
+export interface IDropDownProps {
+  field: any;
+  handleDropdownChange: any;
+}
+
+export class DropDown extends React.Component<IDropDownProps> {
+  itemRenderer(value: string, options: any): React.ReactElement {
+    return (
+      <Button
+        className={DROPDOWN_ITEM_CLASS}
+        onClick={options.handleClick}
+        key={value}
+        text={value}
+      ></Button>
+    );
+  }
+
+  renderCreateOption = (
+    query: string,
+    active: boolean,
+    handleClick: React.MouseEventHandler<HTMLElement>
+  ): React.ReactElement => (
+    <MenuItem
+      icon="add"
+      text={`Create "${query}"`}
+      active={active}
+      onClick={handleClick}
+      shouldDismissPopover={false}
+    />
+  );
+
+  filterDropdown: ItemPredicate<string> = (
+    query,
+    value,
+    _index,
+    exactMatch
```

Comment:
again, is there a way to use partial match instead?

Reply:
I think right now the `exactMatch` variable isn't actually being used, I just needed to take that as an argument for this function. Partial matches work in the dropdown as far as I can tell:
![image](https://user-images.githubusercontent.com/6673460/85623228-b2eecf00-b62d-11ea-952b-841c7e89c5c6.png)